managed/devops/bin/cluster_health.py

```diff
@@ -432,17 +432,17 @@ def check_ysqlsh(self):
         ysqlsh = '{}/bin/ysqlsh'.format(YB_TSERVER_DIR)
         if not self.enable_tls_client:
             user = "postgres"
-            remote_cmd = r'echo "\conninfo" | {} -h {} -p {} -U {}'.format(
+            remote_cmd = "{} -h {} -p {} -U {} -c \"\conninfo\"".format(
                 ysqlsh, self.node, self.ysql_port, user)
         else:
             user = "yugabyte"
-            remote_cmd = r'echo "\conninfo" | {} -h {} -p {} -U {} {}'.format(
+            remote_cmd = "{} -h {} -p {} -U {} {} -c \"\conninfo\"".format(
```

Comment:
This caused a regression with YSQL health checks for ysql authentication enabled universes.